app/jobs/calendar_importer/events/linked_data_event.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module CalendarImporter::Events
+  class LinkedDataEvent < Base
+    attr_reader :uid, :start_time, :end_time, :summary, :description, :location
+
+    def read_value_of(data, field_name)
+      field_data = data[field_name]
+      return unless field_data
+
+      field_data['@value']
+    end
+
+    def parse_timestamp(value)
+      DateTime.parse value.to_s
+    rescue Date::Error
+      nil
+    end
+
+    def initialize(data)
+      super data
+
+      @url = data['url']
+      return if @url.blank?
+
+      @description = data['description']
+      return if @description.blank?
```

Comment:
does this mean it won't import the event if it doesn't have a description?

Reply:
That is correct

Reply:
I think that's too restrictive - we should be importing events permissively and then deciding what to do with them afterwards. I vaguely remember discussing this before and not sure if we made notes?

Reply:
it does sound familiar. its not validated in the model so i will fix that